src/unix/bsd/netbsdlike/netbsd/mod.rs

```diff
@@ -2294,6 +2294,9 @@ extern "C" {
         denominator: ::intmax_t,
         precision: ::size_t,
     ) -> *mut ::c_char;
+    pub fn login(ut: *mut utmp);
+    pub fn logout(line: *mut ::c_char);
+    pub fn logwtmp(line: *mut ::c_char, name: *mut ::c_char, host: *mut ::c_char);
```

Comment:
Seems [they all are `*const`](https://github.com/NetBSD/src/blob/05082e19134c05f2f4b6eca73223cdc6b5ab09bf/include/util.h#L88-L94), aren't they?

Reply:
yes went too fast on this one.

Reply:
You forgot to update `login`.

Reply:
login is correct tough ?

Reply:
ah right I see the signature of login had been changed in the meantime but in the current release it s "mutable"